tests/test_features.py

```diff
@@ -126,13 +126,16 @@ def test_geometry_mask_invert(basic_geometry, basic_image_2x2):
 @pytest.mark.parametrize("geom", [{'type': 'Invalid'}, {'type': 'Point'}, {'type': 'Point', 'coordinates': []}])
 def test_geometry_invalid_geom(geom):
     """An invalid geometry should fail"""
-    with pytest.raises(ValueError) as exc_info, pytest.warns(ShapeSkipWarning):
+    with pytest.warns((RasterioWarning, ShapeSkipWarning)) as record:
         geometry_mask(
```

Comment:
I think it would be a good idea to check the output in this scenario (i.e. shape & filled value).